concourse/scripts/entry.sh

```diff
@@ -130,11 +127,6 @@ function create_fake_gpdb_src() {
         grep -rhw '/usr/local/greenplum-db-devel' -e 'abs_top_srcdir = .*' |\
         head -n 1 | awk '{ print $NF; }')"
 
-    if [ -d "${fake_gpdb_src}" ]; then
```

Comment:
Why these lines were removed?

Reply:
This check works incorrect. Command 
```
$(\
        grep -rhw '/usr/local/greenplum-db-devel' -e 'abs_top_srcdir = .*' |\
        head -n 1 | awk '{ print $NF; }')
```
returns string `/home/gpadmin/gpdb_src`.
This path is path of sources of GPDB. This directory is not empty. Sources are not configured. But this check looks: if directory is not empty than configure is not needed and skip this step.

Reply:
Ok. What does do code block below (lines after configure block)?

Reply:
These lines do useless things:
```
+ local fake_root
++ dirname /home/gpadmin/gpdb_src
+ fake_root=/home/gpadmin
+ mkdir -p /home/gpadmin
+ ln -s /home/gpadmin/gpdb_src /home/gpadmin/gpdb_src
```
At our countainer there is directory `/home/gpadmin/gpdb_src` and these commands do nothing.